test/test_jit.py

```diff
@@ -9972,15 +9993,20 @@ def __init__(self):
 
             # Check there are no Python ops by exporting
             self.assertExportImportModule(module, tensors)
-            create_module.last_graph = module.graph
+            create_script_module.last_graph = module.graph
+            return module(*args)
+
+        # Construct a normal nn module to stay consistent with create_script_module
+        # and make use of a single global rng_state in module initialization
+        def create_nn_module(*args, **kwargs):
```

Comment:
Why is this necessary here?

Reply:
Consider a normal nn module with some random generator functions in its __init__ method: Taking batchnorm as a example, __init__ is calling `reset_parameters`, which have random distribution generated, if we initialize the normal BatchNorm module outside, it will generate a distribution base on a different rng state with ScriptModule, then forward pass will be wrong because it use a different random parameter. So we need to ensure normal nn module and scriptModule share the same rng state even during model initialization. 

This change make every normal nn module and scriptModule create and run only in `self.runAndSaveRng` to share the single state.